Apps/Contoso.Android.Puppet/Constants.cs

```diff
@@ -11,5 +11,6 @@ public static class Constants
         public const string Warning = "Warning";
         public const string Error = "Error";
         public const string StorageSizeKey = "StorageSizeKey";
+        public const string AllowNetworkRequests = "allow-network-requests";
```

Comment:
Let's make it consistent 
```suggestion
        public const string AllowNetworkRequestsKey = "AllowNetworkRequests";
```